frontend/src/metabase/search/components/filters/LastEditedByFilter.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import { t } from "ttag";
+import type { SearchFilterDropdown } from "metabase/search/types";
+import { UserNameDisplay } from "metabase/search/components/UserNameDisplay/UserNameDisplay";
+import { SearchUserPicker } from "metabase/search/components/SearchUserPicker/SearchUserPicker";
+import { convertUserIdToString, parseUserId } from "metabase/search/utils";
+
+export const LastEditedByFilter: SearchFilterDropdown<"last_edited_by"> = {
+  iconName: "person",
+  label: t`Last edited by`,
+  type: "dropdown",
+  DisplayComponent: ({ value: userId }) => (
+    <UserNameDisplay userId={userId} label={LastEditedByFilter.label} />
+  ),
+  ContentComponent: SearchUserPicker,
+  fromUrl: parseUserId,
+  toUrl: convertUserIdToString,
```

Comment:
I just realized `stringify` is such a nice word. No need to update it here, just sharing.

```suggestion
  toUrl: stringifyUserId,
```

Reply:
changed, it's nice to go with `parse` and `stringify` like the json methods